src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/entity/UpdatePayment.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@Table(name = "update_payments_table")
```

Comment:
If there isn't an update payments data table, do we need this to be called update_payments_table? Might be better for it to simply be update_payments

Reply:
changed name to payments